cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/OptimizationOptions.java

```diff
@@ -73,15 +75,10 @@ public OptimizationOptions(Set<String> excludedTopics,
                              boolean isTriggeredByGoalViolation,
                              Set<Integer> requestedDestinationBrokerIds,
                              boolean onlyMoveImmigrantReplicas) {
-    if (excludedTopics == null) {
-      throw new IllegalArgumentException("Excluded topics cannot be null.");
-    } else if (excludedBrokersForLeadership == null) {
-      throw new IllegalArgumentException("Excluded brokers for leadership cannot be null.");
-    } else if (excludedBrokersForReplicaMove == null) {
-      throw new IllegalArgumentException("Excluded brokers for replica move cannot be null.");
-    } else if (requestedDestinationBrokerIds == null) {
-      throw new IllegalArgumentException("Requested destination broker ids cannot be null.");
-    }
+    validateNotNull(excludedTopics, "Excluded topics cannot be null.");
```

Comment:
Since `validateNotNull ` returns the object if it's not null, you can do:

_excludedTopics = validateNotNull(excludedTopics, "Excluded topics cannot be null.");
_excludedBrokersForLeadership = validateNotNull(excludedBrokersForLeadership, "Excluded brokers for leadership cannot be null.");
...